lib/elastic_record/index/documents.rb

```diff
@@ -155,6 +155,16 @@ def search(elastic_query, options = {})
         get url, elastic_query.update('_source' => load_from_source)
       end
 
+      def multi_search(elastic_queries)
+        url = "_msearch"
+
+        queries = elastic_queries.flat_map do |query|
+          ["{}", query.update('_source' => load_from_source).to_json]
```

Comment:
`load_from_source` should not be here. It is not the job of this class to mutate the query and add `load_from_source`. The user of this method should pass that in.